### Задача 1

|  x<sub>i</sub>  |  1  |  2  |  4  |  6  |  7  |
|  ---            | --- | --- | --- | --- | --- |
|  n<sub>i</sub>  |  2  |  3  |  10 |  3  |  2  |

Довірчі інтервали для оцінки середнього квадратичного відхилення:\
s(1-q) < ϭ < s(1+q), для q<1\
0 < ϭ < s(1+q), для q>1

In [17]:
s = 1.79
q = 0.37

In [18]:
interval1 = s * (1 - q)
interval1

1.1277

In [19]:
interval2 = s * (1 + q)
interval2

2.4523

Отже, довірчі інтервали середнього квадратичного відхилення будуть:\
1.1277 < ϭ < 2.4523

### Задача 2

In [20]:
t = 1.96
n = 100
m = 40

Для оцінки з надійністю 0.95 ймовірності настання події А в кожному із 100 незалежних випробувань, якщо подія А відбулась 40 раз, використаємо довірчий інтервал

In [21]:
p1 = (m + t**2 / 2 - t * (m*(n-m) / n + t**2 / 4)** .5) / (t**2 + n)
p1

0.3093997461136028

In [22]:
p2 = (m + t**2 / 2 + t * (m*(n-m) / n + t**2 / 4)** .5) / (t**2 + n)
p2

0.4979992153815975

Довірчий інтервал для настання події А становитиме:\
0.3094 < p < 0.498

### Задача 3

In [23]:
t = 2.58
est_accur = 0.6
sd = 3

Об'єм вибірки, необхідний для інтервальної оцінки математичного сподівання вирахуємо за формулою:

In [24]:
n = t**2 * sd**2 / est_accur**2
n

166.41000000000003